Shippo/Substatus.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Shippo
+{
+    [JsonObject(MemberSerialization.OptIn)]
+    public class Substatus
+    {
+        [JsonProperty(PropertyName = "code")]
+        public string Code;
+
+        [JsonProperty(PropertyName = "text")]
+        public string Test;
```

Comment:
@dochoffiday I'm hoping that another git commit (or empty commit `git commit --allow-empty -m "Commit message here"`) will trigger the github / cirlceci exchange (to kick off the passing tests).  Would it be possible to amend this line to `public string Text;`

Reply:
👍 done